assets/src/scripts/store.js

```diff
@@ -25,6 +39,12 @@ export const Actions = {
                     // Trigger a call to get last year's data
                     dispatch(Actions.retrieveCompareTimeseries(siteno, startTime, endTime));
 
+                    // Update the gage height if gage height is in the data.
+                    const currentGageHeight = getLatestValue(collection, GAGE_HEIGHT_CD);
+                    if (currentGageHeight) {
+                        dispatch(Actions.setGageHeight(currentGageHeight));
+                    }
```

Comment:
I'm wondering if this logic could be done in the flood slider component if `gageHeight === null` rather than here, to keep timeSeries handling separate from the gage height.

Reply:
The problem as I see it is we have two ways of setting the gageHeight. One is retrieving the current time series and the other is with the slider. I'm concerned that if we don't do it as part of the time series retrieval, there will be some nasty conditions where the flood slider has been rendered and used before the timeseries data has finished loading.